src/NexusMods.Icons/IconValue.cs

```diff
@@ -47,7 +47,7 @@ public sealed class IconValue
     
     public SimpleVectorIconImage? SimpleVectorSetter
     {
-        set => Value = new SimpleVectorIcon(value);
+        set => Value = new SimpleVectorIcon(value!);
     }
```

Comment:
Setter should instead be changed to `public SimpleVectorIconImage SimpleVectorSetter` or removed.